newIDE/app/src/UI/CustomSvgIcons/Folder.js

```diff
@@ -2,18 +2,22 @@ import React from 'react';
 import SvgIcon from '@material-ui/core/SvgIcon';
 
 export default React.memo(props => (
-  <SvgIcon {...props} width="24" height="24" viewBox="0 0 24 24" fill="none">
+  <SvgIcon {...props} width="24" height="24" viewBox="0 0 24 24">
     <path
-      fillRule="evenodd"
-      clipRule="evenodd"
-      d="M4 7.75C4 7.33579 4.33579 7 4.75 7H17.25C18.7688 7 20 8.23122 20 9.75V17.25C20 18.7688 18.7688 20 17.25 20H6.75C5.23122 20 4 18.7688 4 17.25V7.75ZM5.5 8.5V17.25C5.5 17.9404 6.05964 18.5 6.75 18.5H17.25C17.9404 18.5 18.5 17.9404 18.5 17.25V9.75C18.5 9.05964 17.9404 8.5 17.25 8.5H5.5Z"
-      fill="currentColor"
+      d="m19.25 17.25v-7.5c0-1.1046-0.8954-2-2-2h-12.5v9.5c0 1.1046 0.89543 2 2 2h10.5c1.1046 0 2-0.8954 2-2z"
+      stroke="currentColor"
+      stroke-linecap="round"
+      stroke-linejoin="round"
+      stroke-width="1.5"
```

Comment:
For the 3 props `stroke-...` you should see warnings in the JS console telling you React does not interpret those attributes and you should use `strokeWidth, ...` instead

Reply:
I don't see the warnings, is it the same on elements within the svg?

Reply:
Here are the errors I see:
<img width="500" alt="image" src="https://github.com/4ian/GDevelop/assets/32449369/3c8e80a2-cacc-4a97-867f-252741005cb5">
<img width="499" alt="image" src="https://github.com/4ian/GDevelop/assets/32449369/b16f1b32-1727-4afb-8863-0da1e3d00b63">
<img width="499" alt="image" src="https://github.com/4ian/GDevelop/assets/32449369/0c4f2229-91e5-4671-b941-a2b111731685">